Makefile

```diff
@@ -160,6 +160,7 @@ test/gui: run-cmake-debug
 	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_top_settings_dlg.tcl
 	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_ipconfigurator.tcl
 	$(XVFB) ./dbuild/bin/pinassignment --replay tests/TestGui/gui_pinassignment.tcl
+	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_task_clean.tcl
```

Comment:
Needs to be added to the valgrind test list.

Reply:
per a conversation with Ravi, this has been changed to a batch style test instead of gui since this is testing the compiler's transition from clean states, not actual gui related events. Gui and Valgrind tests have been removed